script/version.py

```diff
@@ -1,20 +1,39 @@
 #!/usr/bin/env python
 
+import argparse
 import os
 import sys
 
-from lib.config import output_dir, get_brave_version
+from lib.config import CHROMIUM_ROOT, get_brave_version
 
 
 def main():
-  create_version()
+  args = parse_args()
+  create_version(args.target_gen_dir[0])
 
+def parse_args():
+  parser = argparse.ArgumentParser(description='Transpile web-uis')
+  parser.add_argument('-p', '--production',
+                      action='store_true',
+                      help='Uses production config')
+  parser.add_argument('--target_gen_dir',
+                      nargs=1)
+  return parser.parse_args()
 
-def create_version():
-  version_path = os.path.join(output_dir(), 'version')
+
+def create_version(target_gen_dir):
+  version_path = os.path.abspath(os.path.join(CHROMIUM_ROOT, target_gen_dir, '..', 'version'))
```

Comment:
This should probably eventually be moved to the gen dir as well @mbacchi 
This also fixes a bug where it would always use Release dir even if you had a different target specified.